conductr_cli/conduct_acls.py

```diff
@@ -0,0 +1,157 @@
+        display_tcp_acls(log, tcp_acls)
+
+    return True
+
+
+def display_tcp_acls(log, tcp_acls):
+    def tcp_port_value(line):
+        return int(line['tcp_port'])
+
+    data = [
+        {
+            'tcp_port': 'TCP/PORT',
+            'bundle_id': 'BUNDLE ID',
+            'bundle_name': 'BUNDLE NAME',
+            'status': 'STATUS'
+        }
+    ] + sorted([
+        {
+            'tcp_port': tcp_port,
+            'bundle_id': tcp_acl['bundle_id'],
+            'bundle_name': tcp_acl['bundle_name'],
+            'status': tcp_acl['status']
+        }
+        for tcp_acl in tcp_acls
+        for tcp_port in tcp_acl['acl']['tcp']['requests']
+    ], key=tcp_port_value)
+
+    padding = 2
+    column_widths = dict(screen_utils.calc_column_widths(data), **{'padding': ' ' * padding})
+    for row in data:
+        log.screen(
+            '{tcp_port: <{tcp_port_width}}{padding}'
+            '{bundle_id: <{bundle_id_width}}{padding}'
+            '{bundle_name: <{bundle_name_width}}{padding}'
+            '{status: <{status_width}}'.format(**dict(row, **column_widths)).rstrip())
+
+
+def display_http_acls(log, http_acls):
+    http_request_mappings = [
+        {
+            'http_method': http_request_mapping['method'] if 'method' in http_request_mapping else ALL_HTTP_METHOD,
+            'http_rewrite': http_request_mapping['rewrite'] if 'rewrite' in http_request_mapping else '',
+            'http_acl': get_http_acl(http_request_mapping),
+            'http_acl_type': get_http_acl_type(http_request_mapping),
+            'bundle_id': http_acl['bundle_id'],
+            'bundle_name': http_acl['bundle_name'],
+            'status': http_acl['status']
+        }
+        for http_acl in http_acls
+        for http_request_mapping in http_acl['acl']['http']['requests']
+    ]
+
+    def capture_group_count_reverse(line):
+        return len(re.findall('([^\\(\\)]+)', line['http_acl'])) * -1
+
+    http_path_regex_sorted = sorted([
+        request_mapping
+        for request_mapping in http_request_mappings if request_mapping['http_acl_type'] == HTTP_ACL_PATH_REGEX
+    ], key=capture_group_count_reverse)
+
+    def path_depth_reverse(line):
+        return len(line['http_acl'].split('/')) * -1
+
+    http_path_beg_sorted = sorted([
+        request_mapping
+        for request_mapping in http_request_mappings if request_mapping['http_acl_type'] == HTTP_ACL_PATH_BEG
+    ], key=path_depth_reverse)
+
+    http_path_sorted = sorted([
+        request_mapping
+        for request_mapping in http_request_mappings if request_mapping['http_acl_type'] == HTTP_ACL_PATH
+    ], key=path_depth_reverse)
+
+    data = [{
+        'http_method': 'METHOD',
+        'http_acl': 'PATH',
+        'http_rewrite': 'REWRITE',
+        'bundle_id': 'BUNDLE ID',
+        'bundle_name': 'BUNDLE NAME',
+        'status': 'STATUS'
+    }] + http_path_regex_sorted + http_path_beg_sorted + http_path_sorted
```

Comment:
Ordering of the HTTP based ACL will be displayed according to what ConductR's ordering:

* Path Regex - within path regex, it's sorted by the number of capture group in reverse
* Path Begin - within path begin, it's sorted by path depth in reverse
* Path - within path, it's sorted by path depth in reverse